Actors/VisualObjects/VisualObjectsApplicationCSharp/ApplicationManifest.xml

```diff
@@ -2,7 +2,7 @@
 <ApplicationManifest xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ApplicationTypeName="VisualObjectsApplicationType" ApplicationTypeVersion="1.0.0" xmlns="http://schemas.microsoft.com/2011/01/fabric">
   <Parameters>
     <Parameter Name="VisualObjects.WebService_InstanceCount" DefaultValue="-1" />
-    <Parameter Name="VisualObjects.ActorService_PartitionCount" DefaultValue="5" />
+    <Parameter Name="VisualObjects.ActorService_PartitionCount" DefaultValue="2" />
```

Comment:
Why change this?

Reply:
This was a actually a bug. If you see the build script of this application what you will see in the first step is remove the VisualObjectsApplicationCSharp folder "rm -r $DIR/VisualObjectsApplicationCSharp".  The actual application manifest file is to this location is coming from "service-fabric-dotnet-core-getting-started\Actors\VisualObjects\ApplicationManifest.xml" location.  Which is partition count as mentioned above. So I made this change.

Reply:
If we want we can remove this folder itself from tracking.

Reply:
If its not getting used, we should remove the flder